Port  VAD tutorial 06

fayejf opened the pull request:
Add Voice Activity Detection training tutorial 
Patch VAD process data scripts for librosa==0.8.0

Next:
Online Inference VAD notebook waiting for retraining to be finished.

fayejf commented:
emm forgot to merge previous commits. Will squash and merger after review.

okuchaiev commented on tutorials/asr/06_Voice_Activiy_Detection.ipynb:
```diff
@@ -0,0 +1,1151 @@
+    "id": "K_M8wpkwd7d7"
+   },
+   "source": [
+    "# Data Preparation\n",
```
this cell can't be run. Should this be text rather than code cell?

fayejf replied:
emm it indeed is text locally but turns out to 'changed' to code cell in colab... trouble shooting it.

fayejf replied:
fixed!

okuchaiev commented on tutorials/asr/06_Voice_Activiy_Detection.ipynb:
```diff
@@ -0,0 +1,1151 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "speech_data_root = os.path.join(data_folder, 'google_dataset_v2')\n",
```
maybe warn that this could take some time

okuchaiev replied:
actually, how long this should take? running forever on Colab

fayejf replied:
Good catch! It typically won't last long in locally but it running forever on Colab. Suspect it's an I/O issue. Gonna update to have much smaller demo data.

fayejf replied:
@okuchaiev here! to use --demo. need the latest process_vad_data

fayejf commented:
> 1. please change references from 'candidate' branch to 'main' 
      Sure! Fixed it

> 2. I get this error: "process_vad_data.py: error: unrecognized arguments: --demo", removing this parameters seems to work fine
**Please see the above. wait removing this param seems to work fine? I thought it gonna take forever for colab**

> 3. it seemed to have trained for 1 epoch not 5 for me - could you please check?
   oh good catch! fixed!

> 4. Link to '07 Offline_and_Online_VAD_Demo' doesn't exist
    Yeah it serves as a placeholder. Let me just take it down and edit link once it's up.
